src/main/resources/static/css/style.css

```diff
@@ -2944,32 +2944,23 @@ input:checked+.form-slider:before {
 	display: none;
 }
 
-.animate
-{
+.animate {
 	transition: all 0.1s;
-	-webkit-transition: all 0.1s;
 }
 
-.action-button
-{
-	position: relative;
+.action-button {
 	padding: 5px 10px;
 	margin: 0px 5px 5px 0px;
```

Comment:
the `margin` property's numbers define the margin clockwise: top, right, bottom, left. Right now this property says 5px on the right and 5px on the bottom. Did you mean 5px on both bottom and top? That's `5px 0px 5px 0px`.